lang/python/Makefile

```diff
@@ -147,13 +147,13 @@ define Build/InstallDev
 		$(PKG_INSTALL_DIR)/usr/include/python$(PYTHON_VERSION) \
 		$(1)/usr/include/
 	$(CP) \
-		$(HOST_BUILD_PREFIX)/lib/python$(PYTHON_VERSION) \
+		$(HOST_PYTHON_LIB_DIR) \
```

Comment:
I applied the 3 patches manually and python2.7 seems to fail at the installdev stage if I get it right:
```
install -d -m0755 /Openwrt/r7800/tmp/stage-python/usr/lib/python2.7/
install -m0644 ./files/python-package.mk ./files/python-host.mk /Openwrt/r7800/staging_dir/target-arm_cortex-a15+neon-vfpv4_musl-1.1.15_eabi/mk/
cp -fpR /Openwrt/r7800/build_dir/target-arm_cortex-a15+neon-vfpv4_musl-1.1.15_eabi/Python-2.7.12/ipkg-install/usr/include/python2.7 /Openwrt/r7800/tmp/stage-python/usr/include/
cp -fpR /Openwrt/r7800/staging_dir/target-arm_cortex-a15+neon-vfpv4_musl-1.1.15_eabi/host/lib/python /Openwrt/r7800/build_dir/target-arm_cortex-a15+neon-vfpv4_musl-1.1.15_eabi/Python-2.7.12/ipkg-install/usr/lib/libpython2.7.so* /Openwrt/r7800/tmp/stage-python/usr/lib/
cp: cannot stat '/Openwrt/r7800/staging_dir/target-arm_cortex-a15+neon-vfpv4_musl-1.1.15_eabi/host/lib/python': No such file or directory
Makefile:238: recipe for target '/Openwrt/r7800/staging_dir/target-arm_cortex-a15+neon-vfpv4_musl-1.1.15_eabi/stamp/.python_installed' failed
make[2]: *** [/Openwrt/r7800/staging_dir/target-arm_cortex-a15+neon-vfpv4_musl-1.1.15_eabi/stamp/.python_installed] Error 1
```
To me it looks like the cp command gets the first argument without "2.7" appendix. That should be coming from HOST_PYTHON_LIB_DIR.